### DISCUSSION TITLE
A question about the calculation of Stokes velocity drift on the surface

### DISCUSSION THREAD

**Discussion by Agoni00** [NOAA-EMC/WW3#764] [Q&A]
In the program w3iogomd.ftn, we can see the calculation of the Stokes drift speed, but I found that the calculated speed is too small, then I uncommented these two lines and the speed is not too small, so It seems that uncommenting the calculated speed is more accurate,  I want to know why the source program to the two line commented out? See the following figure for the source code details.

![2L8T2`L%)JPC BZ@Q%90S@N](https://user-images.githubusercontent.com/92639723/185793308-f05f7982-2fbb-45ab-a26e-21f162a01788.png)

**Comment by CarstenHansen** [comment]
You also created an issue #776 from this discussion. I suggest that you share your experience in issue #669, where @qingli411 is working in  on making the Stokes tail extension an optional feature.
